hud.lua

```diff
@@ -21,9 +21,13 @@ minetest.register_globalstep(function(dtime)
 		local areaStrings = {}
 
 		for id, area in pairs(areas:getAreasAtPos(pos)) do
-			table.insert(areaStrings, ("%s [%u] (%s%s)")
+			if area.faction_open and (areas.faction_available == false or factions.get_player_faction(area.owner) == nil) then
```

Comment:
Line is very long. Please break it for easier readability in small windows.

Reply:
I've updated the code, although in my opinion readability is worse when code is split across multiple lines